engines/sci/engine/script_patches.cpp

```diff
@@ -3279,9 +3279,9 @@ static const uint16 kq7SubtitleFixSignature3[] = {
 	0x63, 0x28,                 // pToa initialized
 	0x18,                       // not
 	0x31, 0x07,                 // bnt [skip init code]
-	0x38, SIG_SELECTOR16(init), // pushi init ($8e for 2.00, $93 for 1.51)
```

Comment:
I don't really like removing comments like these, that show which values were used in which version of the game.